analytics/src/analytics/integrations/db.py

```diff
@@ -52,6 +52,7 @@ def get_connection_parameters(config: DBSettings) -> dict[str, Any]:
     token = (
         config.password if config.local_env is True else generate_iam_auth_token(config)
     )
+    search_path = "public" if config.local_env is True else "app"
```

Comment:
I'd put this more explicitly in an environment variable in the config class itself.

Something like:
```py
db_schema: str = Field ("app", alias="DB_SCHEMA")
```

Then in local.env, put `DB_SCHEMA=public`

This makes it much easier to adjust or configure.

Reply:
Done! Please see latest commit